crypto/fipsmodule/ecdsa/ecdsa_test.cc

```diff
@@ -192,6 +192,7 @@ TEST(ECDSATest, BuiltinCurves) {
       { NID_secp384r1, "secp384r1" },
       { NID_secp521r1, "secp521r1" },
       { NID_secp160r1, "secp160r1" },
+      { NID_secp256k1, "secp256k1" },
```

Comment:
Nit: In the order below, secp256k1 is checked for before secp160r1.

Reply:
Forgot to change this before I pushed the update. If you don't mind I'd let it be like this?

Reply:
Sure